Source/WebCore/SourcesWPE.txt

```diff
@@ -67,6 +67,8 @@ platform/graphics/ANGLEWebKitBridge.cpp
 platform/graphics/GLContext.cpp
 platform/graphics/PlatformDisplay.cpp
 
+platform/graphics/adwaita/SystemFontDatabaseAdwaita.cpp
```

Comment:
“Adwaita” refers to the default GTK theme for user interface controls, which we do reuse in the WPE port. I guess the idea is using this implementation for the WPE port, given that the GTK port gets its own version of the code. If that's the case, it would make more sense to call if `SystemFontDatabaseWPE.cpp` or even `SystemFontDatabaseDefault.cpp` because there is nothing specific about WPE in the code 🤔

Reply:
Cool, no problem 👍

Reply:
Thanks!